lib/assets/javascripts/cable.js.coffee

```diff
@@ -5,4 +5,8 @@
   PING_IDENTIFIER: "_ping"
 
   createConsumer: (url) ->
+    token = $("meta[name=csrf-token]").attr("content")
```

Comment:
:+1: to both points. Can do `document.querySelector("meta[name=csrf-token]")?.getAttribute("content")` instead.